js/siteorigin-panels/view/live-editor.js

```diff
@@ -19,7 +20,7 @@ module.exports = Backbone.View.extend( {
      * Render the live editor
      */
     render: function(){
-        this.$el.html( this.template() );
+        this.setElement( this.template() );
```

Comment:
In theory, it could be called multiple times, every time the view's model changes. So it might not currently be happening here, but say we want to allow the live editor to preview changes without reloading, then we'd probably need to make changes to the model and re-render the view.

Also, something I missed: It's considered good practice to `return this` at the end of a `render` method.